changelogs/current.yaml

```diff
@@ -55,6 +55,12 @@ minor_behavior_changes:
     the cookie is still accepted, but is planned to be obsoleted in the future.
     This behavior change can be reverted by setting
     ``envoy.reloadable_features.stateful_session_encode_ttl_in_cookie`` to ``false``.
+- area: resource_monitors
+  change: |
+    Changed behavior of the fixed heap monitor to count pages allocated to
+    TCMalloc as free memory if it's not used by Envoy. This change can be
```

Comment:
s/count pages allocated to TCMalloc as free memory if it's not used by Envoy./count unused mapped pages as free memory./

Reply:
Done